packages/maskbook/src/components/InjectedComponents/SetupGuide.tsx

```diff
@@ -166,20 +183,83 @@ const useWizardDialogStyles = makeStyles((theme) =>
     }),
 )
 
+const useStyles = makeStyles((theme: Theme) => {
+    return {
+        root: {
+            alignItems: 'center',
+        },
+        content: {
+            marginRight: 16,
+        },
+        footer: {
+            marginLeft: 0,
+            marginTop: 0,
+            flex: 1,
+            display: 'flex',
+            alignItems: 'center',
+            justifyContent: 'center',
+            flexDirection: 'column',
+        },
+        tip: {},
+    }
+})
+
+interface ContentUIProps {
+    dialogType: SetupGuideStep
+    content?: React.ReactNode
+    footer?: React.ReactNode
+    tip?: React.ReactNode
+}
+
+function ContentUI(props: ContentUIProps) {
+    const classes = useStyles(props)
+    const xsMatch = useMatchXS()
+    const wizardClasses = useWizardDialogStyles()
+    switch (props.dialogType) {
+        case SetupGuideStep.FindUsername:
+            return (
+                <Box display="block">
+                    <Box display={xsMatch ? 'flex' : 'block'}>
+                        <main className={classes.content}>{props.content}</main>
+                        <Hidden only="xs">
+                            <div>{props.tip}</div>
+                        </Hidden>
+                        <footer className={classes.footer}>{props.footer}</footer>
+                    </Box>
+                    <Hidden smUp>
+                        <div>{props.tip}</div>
+                    </Hidden>
+                </Box>
+            )
+
+        default:
+        case SetupGuideStep.SayHelloWorld:
```

Comment:
Why make this case default? I suggest explicitly specify the current mode.

Reply:
ok